src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewDataAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// Properties decorated with <see cref="TempDataAttribute"/> will have their values stored in
+    /// and loaded from the <see cref="ViewFeatures.TempDataDictionary"/>. <see cref="TempDataAttribute"/>
+    /// is supported on properties of Controllers, Razor Pages, and Razor Page Models.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property, Inherited = true, AllowMultiple = false)]
+    public sealed class ViewDataAttribute : Attribute
+    {
+    }
```

Comment:
Another thought, should this allow you to specify whether or not a value is optional? For instance should it blow up with a friendly error if the value is not specified? This can lead to a lot of null refs in views.